src/components/DatePicker/picture-date-picker.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { AdapterDateFns } from "@mui/x-date-pickers/AdapterDateFns";
+import { DatePicker } from "@mui/x-date-pickers/DatePicker";
+import { DatePickerContainer } from "./styles/picture-date-picker.styled";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import { TextField } from "@mui/material";
+
+export const PictureDatePicker = (props) => {
+  const { date, setDate } = props;

```

Comment:
props could be destructured in the component definition:

```suggestion
export const PictureDatePicker = ({ date, setDate }) => {
```